modules/gapi/cmake/Standalone.cmake

```diff
@@ -0,0 +1,30 @@
+if (NOT TARGET ade )
```

Comment:
Please use "lowercase" in file names: standalone.

Also you may put this file one level upper.